include/config/config_compression.h

```diff
@@ -0,0 +1,7 @@
+#ifndef CONFIG_COMPRESSION_H
+#define CONFIG_COMPRESSION_H
+
+/* Size of libdeflates's buffer in kibibytes (yes kib not kb). HackerSM64's implimentation uses a full Mb we use 32 kib*/
+#define KIB 32
+
+#endif
```

Comment:
newline missing there (at the end of the file)

Reply:
also you should probably rename the KIB macro to something more specific to libdeflate, also there's a typo I missed on the comment there ("implimentation")